src/window.ui

```diff
@@ -791,17 +776,38 @@ along with Video Downloader.  If not, see <http://www.gnu.org/licenses/>.
                   <object class="GtkLabel" id="success_msg_wdg">
                     <property name="visible">True</property>
                     <property name="can-focus">False</property>
-                    <property name="label">Download saved in XXX</property>
+                    <property name="halign">center</property>
+                    <property name="label">Download finished</property>
                     <property name="wrap">True</property>
                     <property name="wrap-mode">word-char</property>
                     <property name="xalign">0</property>
```

Comment:
The properties `wrap`, `wrap-mode` and `xalign` can be removed.

Reply:
wrap is still set to true (from a quick visual scan I just did), but I think it's not going to influence the code much (plus I edited it in glade, because it's my favorite way to edit .ui files).